processor.go

```diff
@@ -39,8 +39,9 @@ var errType = reflect.TypeOf((*error)(nil)).Elem()
 
 // RegisterHandler will store the given handler that will be used by the service.
 // WebSocket will then forward requests to "ws://service_name/struct_name"
-// to the given function f, which must be of the following form:
-// func(msg interface{})(ret interface{}, err error)
+// to the given function f, which must be one of the following forms:
+// 1. func(msg interface{})(ret interface{}, err error)
+// 2. func(msg interface{})(ret chan interface{}, err, error)
```

Comment:
I wonder whether we should put both in the same `RegisterHandler` or if we should not have `RegisterHandler` and `RegisterStream`. I have a small preference for the latter.